libs/parallelism/algorithms/include/hpx/parallel/algorithms/detail/find.hpp

```diff
@@ -27,13 +28,14 @@ namespace hpx { namespace parallel { inline namespace v1 { namespace detail {
     }
 
     // provide implementation of std::find_if supporting iterators/sentinels
-    template <typename Iterator, typename Sentinel, typename Pred>
+    template <typename Iterator, typename Sentinel, typename Pred,
+        typename Proj>
     inline constexpr Iterator sequential_find_if(
-        Iterator first, Sentinel last, Pred pred)
+        Iterator first, Sentinel last, Pred pred, Proj&& proj)
```

Comment:
Could you make this consistent, please? Either pass both function objects by value of both using forwarding references (preferred).

Reply:
Sure. I am working on it right now ;)